tests/e2e/cloudprovider/test-lb-service.sh

```diff
@@ -109,6 +151,37 @@ spec:
 EOF
 }
 
+########################################################################
+## Name: create_deployment
```

Comment:
comments here needs to be updated I suppose

Reply:
Thanks, done.